README.md

```diff
@@ -168,305 +116,247 @@ Typical Java threads
 
-Actual name | Description | Run in parallel?
-------------|-------------|------------------
-STUDY | prepare, coordinate idents | No
-ID | identification phase | No
-COORDINATE | Coordinate processing | No
-PRETRANSFORM | before transform hook | Yes
-TRANSFORM | transformation phase | No
-POSTTRANSFORM | after transform hook | Yes
-ANALYZE | analysis/metadata generation | Yes
-VERIFY | verify for output | No
-IO | output | No
-REVIEW |  finish off | No
-
-#### Parsers vs Places
-
-Emissary Places are for processing, transforming, identifying, analyzing data of interest. As items
+### Parsers vs Places
+
+Emissary Places are for identifying, processing, transforming, and analyzing data of interest. 
+
+As items
 are unwrapped and processed the results are all kept together in a "family tree". This structure
-is a List<IBaseDataObject> in the code. The list can be sorted in family order by using the
-emissary.util.ShortNameComparator. 
+is a List\<IBaseDataObject> in the code.
 
-Sometimes, for our own convenience, or for dealing with formats provided by up-stream systems,
-the data arrives packaged in various formats. These containers can have multiple items that
-should not be logically related in a family tree. The Emissary Parser framework deals with this
-situation.
+The Emissary Parser framework is designed to extract information from any data containers/wrappers on the payload that may contain information useful for identification or processing.
 
-Parsers need to make a quick identification of the format of the container. This is called
-out by the emissary.parser.ParserFactory and uses the emissary.parser.DataIdentifier as the
+Parsers need to make a quick identification of the container format (i.e. Apache NiFi). This is called
+out by the emissary.parser.ParserFactory and uses the [emissary.parser.DataIdentifier](src/main/java/emissary/parser/DataIdentifier.java) as the
 engine to perform the identification. The engine is configured by name and can be replaced
 by anything that meets the required interface. The name of the format identified should
 have parsers that meet the SessionParser interface configured into the factory.
 
 Once the container is identified, items are parsed out of it, built into IBaseDataObject instances
 by the SessionParser appropriate for that data type, and handed over to MobileAgent instances
-for processing as they become available from the pool. It is important to have in mind that the
+for processing as they become available from the pool. 
+
+It is important to have in mind that the
 Parser framework is operating on the single-threaded side of the system and handing items over to the
-multi-threaded (MobileAgent) side of the system. This implies that the parsers are fast enough
+multi-threaded (MobileAgent) side of the system. This necessitates that the parsers are fast enough
 to keep the number of configured threads busy.
 
 ## Development
 
 ### Requirements
 
-#### Linux or MacOSX operating system
```

Comment:
Made into a preference (see below) instead of a requirement below. Windows now has a unix bash terminal and development there _is_ possible